ros/src/models/chrono/ros_chrono/src/data/vehicle/paths/switch_lane.txt

```diff
@@ -1,5 +1,5 @@
 4 3
-     56.0000    -125.0000       0.1000
-   -125.0000    -125.0000       0.1000
-   -125.0000    -120.0000       0.1000
+     56.0000    -122.0000       0.1000
```

Comment:
can we put this as a YAML file? and then upload it to a ROSparam?

Reply:
Okay

Reply:
thanks